lib/rules/one-var.js

```diff
@@ -62,20 +120,46 @@ module.exports = function(context) {
         "FunctionDeclaration": startFunction,
         "FunctionExpression": startFunction,
         "ArrowFunctionExpression": startFunction,
+        "BlockStatement": startBlock,
```

Comment:
So @sebmck fixed the babel issue in babel v5.1.11. I know that escope is pinned to babel < 5. Which is the better path, to move expect the error in this test for now, with a todo comment for when escope updates babel, or to just hold this PR until escope updates?